addons/web/static/src/views/list/list_renderer.js

```diff
@@ -184,6 +187,37 @@ export class ListRenderer extends Component {
         });
     }
 
+    /**
+     * @returns {Array} of name of 'Properties' fields
+     */
+    getPropertiesFieldNames() {
+        const propertiesFieldNames = [];
+        for(const record of Object.values(this.props.list.records)) {
```

Comment:
They may have different values of properties if we're looking at, say, 'all tickets'. I'll be looking for a way to make that cheaper if possible though. lna also proposed we just don't show properties when there are properties coming from different teams since it could be a bit too much if there are many teams, but I think it'd be cool to have the option.